app/server/plugins/games/global/SandalePlugin.ts

```diff
@@ -102,6 +102,8 @@ export class SandalePlugin extends GlobalPlugin {
         if (message.indexOf('/message') === 0 && message.split(' ').length > 2 && this.isSandaled(connection.session.identifier)) {
             // Replace the message with the sandale smiley
             return '/message 👞';
+            this.setSandaled(identifier, false);
```

Comment:
Also `identifier` doesn't exist there it's `connection.session.identifier`

```suggestion
            this.setSandaled(connection.session.identifier, false);
```